internal/hardware/retrieve.go

```diff
@@ -47,43 +48,42 @@ func (c *Client) Retrieve() *cobra.Command {
 			header := []string{"ID", "Facility", "Plan", "Created"}
 
 			inc := []string{}
+			exc := []string{}
+			var query, state, provisionable string
 
 			// only fetch extra details when rendered
 			switch c.Servicer.Format() {
 			case outputs.FormatJSON, outputs.FormatYAML:
 				inc = append(inc, "project", "facility", "device")
 			}
 
-			listOpt := c.Servicer.ListOptions(inc, nil)
-
 			if hardwareReservationID == "" && projectID == "" {
 				return fmt.Errorf("either id or project-id should be set")
 			}
 
 			cmd.SilenceUsage = true
 			if hardwareReservationID != "" {
-				getOpts := &packngo.GetOptions{Includes: listOpt.Includes, Excludes: listOpt.Excludes}
-				r, _, err := c.Service.Get(hardwareReservationID, getOpts)
+				r, _, err := c.Service.FindHardwareReservationById(context.Background(), hardwareReservationID).Include(inc).Exclude(exc).Execute()
```

Comment:
This also needs to be wired up to the `Includes()` and `Excludes()` functions as well

Reply:
updated